Add deployment monitor to dbt cloud deploy page (#3199)
dbt-labs/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes/09-April-2023/deployment-monitor.md

---

> title: "The new Deployment monitor page has improved filtering and sorting capabilities and provides greater visibility to your job run health"

Author: ```suggestion
title: "Use the Deployment monitor page for greater visibility to your job run health"
```
